gas/testsuite/ChangeLog.COREV

```diff
@@ -1,3 +1,188 @@
+2022-07-29  Charlie Keaney  <charlie.keaney@embecosm.com>
+
+	* gas/riscv/zc-zcmb-fail-arch-conflict-0.l: Fixed typo in error message.
+	* gas/riscv/zc-zcmb-fail-arch-conflict-1.l: Fixed typo in error message.
+	* gas/riscv/zc-zcmpe-fail-arch-conflict-1.l: Fixed typo in error message.
+	* gas/riscv/zc-zcmt-jalt.d: Added test for successful assembly of instruction.
+	* gas/riscv/zc-zcmt-jalt.s: Added test for successful assembly of instruction.
+	* gas/riscv/zc-zcmt-jt.d: Added test for successful assembly of instruction.
+	* gas/riscv/zc-zcmt-jt.s: Added test for successful assembly of instruction.
+	* gas/riscv/zc-zcb-c-lbu-range-fail.d: Added test for failure on invalid immediate offset outside of range.
```

Comment:
These should be flowed to 80 columns.

Also for consistency, I see some `Likewise`s, I would use these throughout if it is the same thing.

Reply:
Fixed in next patch.